### PR TITLE
Update and Delete for Organizations 

### PR DESCRIPTION
**Pull request by misstonbon**
Hooks up update and delete organizations, refactors components.

### CODE REVIEW

File: __tests__/reducers/organization.test.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ describe('Organization Reducer', () => {
 		expect(expectedState).to.deep.equal(actualState)
 	})
 
-	it('should handle FETCHED_ORGANIZATIONS.', () => {
+	it.only('should handle FETCHED_ORGANIZATIONS.', () => {
```
Comment (tayhalla) [inline __tests__/reducers/organization.test.ts]: @misstonbon eliminate the `only`

File: __tests__/reducers/organization.test.ts [hunk 2]
```diff
@@ -45,7 +45,7 @@ describe('Organization Reducer', () => {
 		expect(expectedState).to.deep.equal(actualState)
 	})
 
-	it('should handle UPDATED_ORGANIZATION.', () => {
+	it('should handle UPDATED_ORGANIZATION.', async () => {
```
Comment (tayhalla) [inline __tests__/reducers/organization.test.ts]: @misstonbon either all of these need to follow a async/await pattern or none. Grab me if this is confusing

File: src/components/Organization/index.tsx [hunk 3]
```diff
@@ -31,15 +36,70 @@ interface IConnectedActions {
...
+	public createNewOrg = (e: React.MouseEvent<Button>) => {
+		e.preventDefault()
+		this.props.organizationActions.createOrganization(this.state.name)
+		this.setState({name: ''})
+		this.closeModal()
+	}
+
+	public editOrg = () => {
+		// tslint:disable-next-line:no-console
+		console.log('props', this.props)
```
Comment (tayhalla) [inline src/components/Organization/index.tsx]: Remove the console

File: src/components/Organization/index.tsx [hunk 4]
```diff
@@ -84,31 +143,62 @@ class OrganizationViewComponent extends React.Component<Props> {
...
+
+	public showForm = () => {
+		return (
+			<Form
+				handleInputChange={this.handleInputChange}
+			/>
+		)
+	}
+
+	// public launchNewOrgModal = () => {
```
Comment (kcoleman731) [inline src/components/Organization/index.tsx]: commented code!

File: src/components/Organization/index.tsx [hunk 5]
```diff
@@ -84,31 +143,62 @@ class OrganizationViewComponent extends React.Component<Props> {
...
+	// 	})
+	// }
+
+	/**
+	 * Render
+	 */
 	public render() {
 		return (
-			<div className=''>
+			< div className = '' >
```
Comment (kcoleman731) [inline src/components/Organization/index.tsx]: should we give this a class?

**Pull request merged**